measures/measures-data.json

```diff
@@ -1362,7 +1362,7 @@
     "metricType": "boolean",
     "measureId": "IA_PM_3",
     "cehrtEligible": false,
-    "description": "Participating in a Rural Health Clinic (RHC), Indian Health Service Medium Management (IHS), or Federally Qualified Health Center in ongoing engagement activities that contribute to more formal quality reporting in line with Section 1848(q)(2)(B)(iii) of the Act that requires the Secretary to give consideration to the circumstances of practices located in rural areas and geographic HPSAs.  Rural Health Clinics would be included in that definition for consideration of practices in rural areas.",
+    "description": "Participating in a Rural Health Clinic (RHC), Indian Health Service (IHS), or Federally Qualified Health Center in ongoing engagement activities that contribute to more formal quality reporting, and that include receiving quality data back for broader quality improvement and benchmarking improvement which will ultimately benefit patients. Participation in Indian Health Service, as an improvement activity, requires MIPS eligible clinicians and groups to deliver care to federally recognized American Indian and Alaska Native populations in the U.S. and in the course of that care implement continuous clinical practice improvement including reporting data on quality of services being provided and receiving feedback to make improvements over time.",
```

Comment:
@marimiyachi Looks like there actually was a small description and title update from the cms endpoints we curl from.